### ISSUE TITLE
Gem doesn't work with Ruby 2.5.0

### ISSUE THREAD

**Issue by En1gma99** [toy/pHash#14]
Traceback (most recent call last):
	8: from task.rb:1:in `<main>'
	7: from /home/lincalinc/.rbenv/versions/2.5.0/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:39:in `require'
	6: from /home/lincalinc/.rbenv/versions/2.5.0/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:135:in `rescue in require'
	5: from /home/lincalinc/.rbenv/versions/2.5.0/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:135:in `require'
	4: from /home/lincalinc/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/pHash-1.2.0/lib/phash.rb:3:in `<top (required)>'
	3: from /home/lincalinc/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/pHash-1.2.0/lib/phash.rb:63:in `<module:Phash>'
	2: from /home/lincalinc/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/ffi-1.9.25/lib/ffi/library.rb:99:in `ffi_lib'
	1: from /home/lincalinc/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/ffi-1.9.25/lib/ffi/library.rb:99:in `map'
/home/lincalinc/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/ffi-1.9.25/lib/ffi/library.rb:145:in `block in ffi_lib': Could not open library 'pHash': pHash: cannot open shared object file: No such file or directory. (LoadError)
Could not open library 'libpHash.so': libpHash.so: cannot open shared object file: No such file or directory

**Comment by toy** [comment]
I assume the problem was with ffi not been able to find `libpHash.so` automatically, so I've added message about `PHASH_LIB` to exception.

**Comment by Nakilon** [comment]
macOS High Sierra, native Ruby 2.3.3
```
irb(main):002:0> ENV["PHASH_LIB"]
=> "/usr/local/lib/libpHash.dylib"
irb(main):003:0> require "phash"
LoadError: Specify path to pHash library using PHASH_LIB environment variable. Could not open library 'pHash': dlopen(pHash, 0x0005): closured error.
```

**Comment by toy** [comment]
@Nakilon Sadly only today I've noticed that I accidentally broke usage of `PHASH_LIB` environment variable in `v1.2.1` and released a fix in `v1.2.2` just now.
